documentation/meta/traffic/runbooks/identifying-and-blocking-traffic-anomalies.md

```diff
@@ -0,0 +1,163 @@
+  Openverse browser (and usually looks something like
+  `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/{webkit-version} (KHTML, like Gecko) Chrome/{real-chrome-version} Safari/{webkit-version}`),
+  no single user agent should take up more than 10% of total traffic. Excluding
+  these with a `Source user agent does not contain Mozilla` or
+  `Source browser equals Unknown` filter in the analytics UI can reveal other
+  user agents from things like social media platforms and popular http request
+  software libraries, like `Go-http-client/2.0` or `axios/0.21.4`, as examples.
+
+- IP addresses: A unique identifier for a single machine or user.
+
+  Typically, a single IP address makes up less than 1% of total Openverse
+  requests in a given timeframe. Larger IPs which exceed this 1% figure
+  typically belong to "good" web crawlers from places like Google, Bing, and
+  Dotbot, as examples. Traffic exceeding 1% which isn't from one of these
+  crawlers could suggest that a user on the ASN is scraping Openverse or
+  accessing Openverse programmatically in an aggressive way.
+
+#### Steps
+
+1.  Using
+    [the view from the previous task](#1-identify-the-timeframe-of-the-traffic-anomalies),
+    scroll down and look for IP Addresses, Source ASNs, or Source User Agents
+    which exceed the typical usage thresholds defined above. Create a list of
+    these traffic sources.
+
+```{note}
+Cloudflare will only show the 5 largest of each category; greater than 5 disproportionately-large items of any category would be unprecedented and surprising in Openverse's history.
+```
+
+````{warning}
+If you haven't found anything, it's unlikely that this situation is connected to malicious traffic or specific actors. Instead, it is more likely to be a normal increase in traffic to Openverse. At this point, it's more likely that a change in code or infrastructure has caused a performance drop, exposed by the higher traffic, or that the traffic has grown so much that we need to allocate additional server resources. Consider recent Openverse marketing efforts or WordPress events if the traffic looks organic and we need to attribute it to something.
+
+2.  Take the list of suspicious traffic source IDs and go to the
+    `Security => Bots` section of Cloudflare. Here, you'll see more lists of IP
+    Addresses, ASNs, and User Agents below a graph of known bot traffic. It's
+    very likely you'll see items from your lists in these graphs.
+
+3.  Use the "add filter" button and filter by your listed resources
+    individually. Write down which of the identified resources are comprised of
+    the most `Automated` and `Likely Automated` traffic.
+
+```{warning}
+If an ASN has a significant amount of human traffic, it can indicate this ASN is an ISP rather than a hosting company or company which offers a Virtual Private Network. Blocking these types of ASNs can disrupt service to human users.
+````
```

Comment:
This looks like it's not quite opened/closed properly, which is making the render look odd. Specifically (suggestions are hard WRT escaping the triple backticks) the first warning here is 4 backticks and doesn't appear to be closed at all.
![image](https://user-images.githubusercontent.com/10214785/235776617-33e25588-830e-42ca-ba63-f99eced635d4.png)